packages/react-avatar/src/components/Image/useImage.tsx

```diff
@@ -1,15 +1,13 @@
 import * as React from 'react';
 import { ImageProps, ImageState } from './Image.types';
-import { getSlots, makeMergeProps } from '@fluentui/react-compose/lib/next/index';
+import { getSlots, mergeProps } from '@fluentui/react-compose/lib/next/index';
 
 export const renderImage = (state: ImageState) => {
   const { slots, slotProps } = getSlots(state);
 
   return <slots.root {...slotProps.root} />;
 };
 
-const mergeProps = makeMergeProps<ImageState>();
```

Comment:
TODO: Add tests to show why we cannot rely on a static list of props to deeply merge.